blackboxopt/optimizers/botorch_base.py

```diff
@@ -390,7 +391,11 @@ def report(self, evaluations: Union[Evaluation, Iterable[Evaluation]]) -> None:
         """A simple report method that conditions the model on data.
         This likely needs to be overridden for more specific BO implementations.
         """
-        _evals = [evaluations] if isinstance(evaluations, Evaluation) else evaluations
+        _evals = (
+            [evaluations]
+            if isinstance(evaluations, Evaluation)
+            else sort_evaluations(evaluations)
+        )
```

Comment:
Switched to that in 72d866c

Reply:
Keeping it in `report()` would have the two benefits of a) having it in sync with other optimizers implementations and b) it might get accidentally called by a child implementation, even if not needed (the sorting add some overhead). But I don't have a strong opinion here and trust your judgement. :-)

Reply:
My impression was that the sorting overhead is negligible compared to e.g. re-fitting a GP, is that a misconception?

Reply:
I would assume that the sorting is always negligible because it's sorting a few python objects. Any serious optimizer will create way more overhead than that. I would be happy to have it in the base class.